plugins/viewableFiles/pdfJsViewer/PdfJsViewerPlugin.inc.php

```diff
@@ -1,21 +1,21 @@
 <?php
 
 /**
- * @file plugins/viewableFiles/pdfSubmissionFile/PdfSubmissionFilePlugin.inc.php
+ * @file plugins/viewableFiles/pdfJsViewer/PdfJsViewerPlugin.inc.php
  *
  * Copyright (c) 2014-2015 Simon Fraser University Library
  * Copyright (c) 2003-2015 John Willinsky
  * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
  *
- * @class PdfSubmissionFilePlugin
- * @ingroup plugins_viewableFiles_pdfSubmissionFile
+ * @class PdfJsViewerPlugin
+ * @ingroup plugins_viewableFiles_pdfJsViewer
```

Comment:
Viewable files makes me think of plugins that implements files that can be viewable... maybe files viewer? Anyway, minor.